### DISCUSSION TITLE
Wi-Fi, Airplane Mode & Hotspot Icons Missing

### DISCUSSION THREAD

**Discussion by ATCJas** [valinet/ExplorerPatcher#1979] [Q&A]
Hi. I just updated to the latest build and lost these icons. Is there any way to get them back without uninstalling EP?

![image](https://github.com/valinet/ExplorerPatcher/assets/112551690/23cc8f44-6bd8-4c9c-b267-00060075e92d)

**Comment by mmschmieg** [comment]
I usually don't run my machines on WiFi, but I just checked and I'm missing them as well.

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.

**Reply by mmschmieg** [reply]
![Screenshot 2023-09-03 1](https://github.com/valinet/ExplorerPatcher/assets/128321659/f811f436-8379-46f7-a4b1-de27d3b17679)

![Screenshot 2023-09-03 2](https://github.com/valinet/ExplorerPatcher/assets/128321659/775a0131-288f-453f-95e8-8cb47488ba16)

![Screenshot 2023-09-03 3](https://github.com/valinet/ExplorerPatcher/assets/128321659/58f6d6d7-d936-49ad-8f86-078bd4e49dcd)

![Screenshot 2023-09-03 4png](https://github.com/valinet/ExplorerPatcher/assets/128321659/c49fa090-8f8a-4cde-9586-074efd083c1b)

![Screenshot 2023-09-03 6](https://github.com/valinet/ExplorerPatcher/assets/128321659/b54637f0-619a-40ac-ba85-e540f8bfebb2)

![Screenshot 2023-09-03 7](https://github.com/valinet/ExplorerPatcher/assets/128321659/1316ca4b-8629-4db5-bf7f-8447283abe33)

![Screenshot 2023-09-03 8](https://github.com/valinet/ExplorerPatcher/assets/128321659/5f6a958d-edd0-49cb-832e-ce40c1022550)

![Screenshot 2023-09-03 9](https://github.com/valinet/ExplorerPatcher/assets/128321659/2eb32dc0-b74e-4670-ae39-da89cc0108d6)

![Screenshot 2023-09-03 10](https://github.com/valinet/ExplorerPatcher/assets/128321659/d4209e24-cd1c-4daf-ad00-4965de12f773)

Hope this helps.

Edit Addendum: I checked again after entering all of the above and, now, when I enable the WiFi, the icons are back. I haven't changed anything, either in Windows or in EP, so I have no idea what caused the original glitch.

**Reply by ATCJas** [reply]
Here are the details:

![image](https://github.com/valinet/ExplorerPatcher/assets/112551690/ebf068a7-48b1-4d79-a178-9e6714eddd6c)

![image](https://github.com/valinet/ExplorerPatcher/assets/112551690/d6ed22b3-6013-49e6-a77e-7a0315645000)

![image](https://github.com/valinet/ExplorerPatcher/assets/112551690/22bb73a4-615b-4740-8b03-68967216c17e)

![image](https://github.com/valinet/ExplorerPatcher/assets/112551690/fbcd89cf-ed2f-452a-a5f4-5d60fccbb1d4)

![image](https://github.com/valinet/ExplorerPatcher/assets/112551690/626f97e4-0689-48af-b8fc-47fb73d3bf6a)

![image](https://github.com/valinet/ExplorerPatcher/assets/112551690/715e38fb-1803-4cff-aebc-24e56db87081)

![image](https://github.com/valinet/ExplorerPatcher/assets/112551690/94691de8-5988-43ff-bbf3-a8fb6bd38689)

![image](https://github.com/valinet/ExplorerPatcher/assets/112551690/104baf70-432b-42dc-8d34-27a81747839a)

**Reply by pyrates999** [reply]
@mmschmieg thank you.

@ATCJas That is a preview update: https://support.microsoft.com/en-us/topic/august-22-2023-kb5029351-os-build-22621-2215-preview-9af25662-083a-43f5-b3a7-975fe25cc692

Uninstall it if you want to use explorer patcher and don't install preview updates in the future: https://github.com/valinet/ExplorerPatcher/wiki/Symbols

In the settings app, untick "Get the latest updates as soon as available" so that preview updates are not automatically installed.

All preview updates are then shown as download and install and are never automatically installed if you have the above unticked.

Preview updates are also not fully tested by Microsoft and thus could have bugs.

Preview updates are not supported by EP.

**Reply by ATCJas** [reply]
Whoops... sorry. Missed uploading weather and updates tabs.

![image](https://github.com/valinet/ExplorerPatcher/assets/112551690/0cd82d51-e7f5-40e7-8b8d-e30fbbb4505b)

![image](https://github.com/valinet/ExplorerPatcher/assets/112551690/bc96d918-131c-491a-b78b-739d88ffe66d)

I dont have pre-releases activated.

![image](https://github.com/valinet/ExplorerPatcher/assets/112551690/592586e4-a508-43dd-a59e-ce527dd1b4c5)

**Reply by pyrates999** [reply]
@ATCJas you need to uninstall the last preview update for windows 11.  Here's the article on it:

https://support.microsoft.com/en-au/topic/august-22-2023-kb5029351-os-build-22621-2215-preview-9af25662-083a-43f5-b3a7-975fe25cc692

So uninstall kb5029351 and reboot.

Then since you have that option off in settings, you will see preview updates as download and install, but you must resist to install them.  You need to ignore them.

**Reply by ATCJas** [reply]
Okay, thank you!
